fancy-explosions/BangazonSite
# Finished issue 1

Fixes Issue #1 

## Type of change

Please delete options that are not relevant.



# Testing Instructions

git fetch --all
git checkout js-issue1
run it
check product Categories
make sure showind the categories


# Checklist:
